### PR TITLE
Feature/tfn 391 uat improve hint texts

### PR DESCRIPTION
**Pull request by gilesdrinkall**
# Description

-   To make small changes to the UI on the back of feedback received as part of user research/GDS assessment.

# Testing instructions

-   Run 'make' in the fdbt-dev repo to view the changes to the site locally.

# Type of change

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [X] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

# Checklist:

-   [X] Able to run pr locally
-   [X] Lighthouse performed
-   [X] Followed acceptance criteria
-   [X] My code follows the style guidelines of this project
-   [X] I have performed a self-review of my own code
-   [ ] I have made corresponding changes to the documentation if applicable
-   [ ] I have added tests that prove my fix is effective or that my feature works

Screenshot 2020-05-05 at 17 14 12

### CODE REVIEW

File: src/pages/howManyStages.tsx [hunk 1]
```diff
@@ -49,10 +52,6 @@ const HowManyStages = ({ errors = [] }: HowManyStagesProps): ReactElement => {
                                         >
                                             20 fare stages or less
```
[the remark this answers is not in the document]
Comment (gilesdrinkall) [reply src/pages/howManyStages.tsx:53]: Noe fixed - gone for 'fare stages'.

File: src/pages/howManyStages.tsx [hunk 2]
```diff
@@ -26,10 +26,13 @@ const HowManyStages = ({ errors = [] }: HowManyStagesProps): ReactElement => {
...
                         <fieldset className="govuk-fieldset" aria-describedby="selection-hint">
                             <legend className="govuk-fieldset__legend govuk-fieldset__legend--xl">
-                                <h1 className="govuk-fieldset__heading">What is the size of your fares triangle?</h1>
+                                <h1 className="govuk-fieldset__heading">
+                                    How many fares stages does the service have?
+                                </h1>
                             </legend>
                             <span className="govuk-hint" id="selection-hint">
-                                You&apos;ll need to upload a CSV if your fares triangle has more than 20 fares stages.
+                                If the service has more than 20 stages fares stages you will be required to upload a csv
```
[the remark this answers is not in the document]
Comment (gilesdrinkall) [reply src/pages/howManyStages.tsx:34]: Now fixed.

File: src/pages/chooseStages.tsx [hunk 3]
```diff
@@ -12,13 +12,13 @@ const ChooseStages = (): ReactElement => (
                     <fieldset className="govuk-fieldset" aria-describedby="page-heading">
                         <legend className="govuk-fieldset__legend govuk-fieldset__legend--xl">
                             <h1 className="govuk-fieldset__heading" id="page-heading">
-                                How many fare stages are there in your fares triangle?
+                                How many fare stages does the service have?
```
[the remark this answers is not in the document]
Comment (gilesdrinkall) [reply src/pages/chooseStages.tsx:15]: Now fixed - gone for 'fare stages'.

File: src/pages/chooseStages.tsx [hunk 4]
```diff
@@ -12,13 +12,13 @@ const ChooseStages = (): ReactElement => (
...
                             </h1>
                         </legend>
 
                         <div className="govuk-form-group">
                             <label className="govuk-hint" htmlFor="width-2">
-                                Number of fare stages (1-20) e.g. 3
+                                Enter the number of fare stages between 1 - 20 (for example 3)
```
[the remark this answers is not in the document]
Comment (gilesdrinkall) [reply src/pages/chooseStages.tsx:21]: Now fixed - gone for 'fare stages'.

File: src/pages/matching.tsx [hunk 5]
```diff
@@ -44,7 +44,10 @@ const Matching = ({ userFareStages, stops, service, error }: MatchingProps): Rea
                         </span>
```
[the remark this answers is not in the document]
Comment (gilesdrinkall) [reply src/pages/matching.tsx:44]: Merged develop into branch to get new pages. All matching pages now have this Traveline hint text.

File: src/pages/csvUpload.tsx [hunk 6]
```diff
@@ -19,16 +19,16 @@ const CsvUpload = (uploadProps: UserDataUploadsProps): ReactElement => (
                 {...uploadProps}
```
[the remark this answers is not in the document]
Comment (gilesdrinkall) [reply src/pages/csvUpload.tsx:19]: csvZoneUpload page now has same changes as the csvUpload page.

File: src/pages/howManyProducts.tsx [hunk 7]
```diff
@@ -30,11 +30,11 @@ const HowManyProducts = ({ inputCheck, errors }: HowManyProductProps): ReactElem
                     <fieldset className="govuk-fieldset" aria-describedby="page-heading">
                         <legend className="govuk-fieldset__legend govuk-fieldset__legend--xl">
                             <h1 className="govuk-fieldset__heading" id="page-heading">
-                                How many products do you have for this zone or selected services?
+                                How many period tickets do you have for the selected services?
                             </h1>
                         </legend>
                         <label className="govuk-label" htmlFor="numberOfProducts">
```
[the remark this answers is not in the document]
Comment (gilesdrinkall) [reply src/pages/howManyProducts.tsx:36]: Changed the className to 'govuk-hint'.

File: src/pages/singleDirection.tsx [hunk 8]
```diff
@@ -39,6 +39,9 @@ const SingleDirection = ({ operator, lineName, service }: DirectionProps): React
                                 selectNameID="directionJourneyPattern"
                                 journeyPatterns={service.journeyPatterns}
                             />
+                            <span className="govuk-hint hint-text" id="traveline-hint">
+                                This data is taken from the Traveline National Dataset
```
Comment (ljones92) [inline src/pages/singleDirection.tsx:43]: Should we also add this to the returnDirection page?

Comment (gilesdrinkall) [reply src/pages/singleDirection.tsx:43]: Added that line twice - under the dropdown for both Out bound and Inbound - not sure what you think?

Comment (ljones92) [reply src/pages/singleDirection.tsx:43]: I think just once under the second dropdown would be enough

Comment (gilesdrinkall) [reply src/pages/singleDirection.tsx:43]: Now shows just once under the second dropdown.

**Pull request merged**